GameServerLib/GameObjects/AttackableUnits/AI/ObjAIBase.cs

```diff
@@ -138,6 +145,9 @@ public class ObjAiBase : AttackableUnit, IObjAiBase
                         Spells[i] = new Spell.Spell(game, this, CharData.SpellNames[i], (byte)i);
                     }
                 }
+                //Passive
+                Spells[(int)SpellSlotType.PassiveSpellSlot] = new Spell.Spell(game, this, CharData.PassiveData.PassiveLuaName, (int)SpellSlotType.PassiveSpellSlot);
```

Comment:
`CharData.PassiveData.PassiveLuaName` is used here, but right after we have ` if (!string.IsNullOrEmpty(CharData.PassiveData.PassiveLuaName))`. Is it defined or not?

Reply:
The purpose is so that any spell not defined in the content data gets assigned BaseSpell.